pulsar-io/twitter/src/main/java/org/apache/pulsar/io/twitter/TwitterFireHose.java

```diff
@@ -71,7 +72,7 @@
 
     @Override
     public void open(Map<String, Object> config, SourceContext sourceContext) throws IOException {
-        TwitterFireHoseConfig hoseConfig = TwitterFireHoseConfig.load(config);
+        TwitterFireHoseConfig hoseConfig = IOConfigUtils.loadWithSecrets(config, TwitterFireHoseConfig.class, sourceContext);
```

Comment:
should this be gated with a config parameter? What if I'm just running a simple connector like twitter on my laptop without having any need for secrets?

Reply:
@srkukarni users can still pass the the "sensitive" fields as they have before.  This won't break that.